MSTest-Playwright-BrowserStack/.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "browserstack-sdk": {
+      "version": "1.12.2",

```

Comment:
Please remove this browserstack-sdk tool, not needed.
`"tools": {}`

Reply:
Sure